### PR TITLE
HintikkaXOData

### PR DESCRIPTION
**Pull request by TuomasBorman**
https://github.com/microbiome/mia/issues/279

All SEs are converted into TreeSEs

### CODE REVIEW

**Review by antagomir — APPROVED** [review]
Seems good! Could be merged.

File: R/mia.R [hunk 1]
```diff
@@ -157,4 +157,42 @@ NULL
 #' \url{DOI:10.7717/peerj.13075/supp-2}
 "peerj13075"
 
-
+#' Multiomics dataset from a rat experiment studying effect of fat and prebiotics in diet
```
Comment (antagomir) [inline R/mia.R:165]: Should we somewhere in this manpage mention that the data is provided in TreeSE format?

**Comment by antagomir** [comment]
TreeSE format and having this readily available in a package seems to work better than online downloads via microbiomeDataSets.

If it could be easily done I would change examples using this data set into this mia version in mia, miaViz, OMA.

File: R/mia.R (hunk 1 above)
Comment (TuomasBorman) [reply R/mia.R:165]: Added format tag to all datasets

**Comment by TuomasBorman** [comment]
Hmm, yes 

> TreeSE format and having this readily available in a package seems to work better than online downloads via microbiomeDataSets.
> 
> If it could be easily done I would change examples using this data set into this mia version in mia, miaViz, OMA.

Changed data from mia examples

--> yes we should also change the data from other places (I think we have just used hintikka data in OMA, and just in couple of places... at least I hope)

**Review by antagomir — APPROVED** [review]
ok to merge

**Pull request merged**